charts/mainflux/values.yaml

```diff
@@ -11,6 +11,21 @@ defaults:
   replicaCount: 3
   natsPort: 4222
   jaegerPort: 6831
+  rabbitmqPort: 5672
+
+queueSystem:
+  natsQueue: true
+  rabbitmqQueue: false
```

Comment:
What happens if both of these are set to true?

Reply:
In this case, we would have both of them running. We need to set only one to true at each time.

Reply:
Also, we cannot use only one boolean, as in the condition statement in Chart.yaml we cannot use the negation of a condition.